src/main/java/com/thevoxelbox/voxelsniper/util/BlockHelper.java

```diff
@@ -320,7 +316,7 @@ public static void paint(final Player p, final boolean auto, final boolean back,
         if (bestMatch != null) {
             if (auto) {
                 try {
-                    final int i = bestMatch.getArt().getId() + (back ? -1 : 1) + Art.values().length % Art.values().length;
+                    final int i = bestMatch.getArt().getId() + (back ? -1 : 1);

```

Comment:
```suggestion
                    final int i = (bestMatch.getArt().getId() + (back ? -1 : 1) + Art.values().length) % Art.values().length;
```

I think this actually has a purpose here, only it was not executed correctly: What would happen, when bestmatch is the last entry, and I add 1, we would probably get null, or an error. Same when bestmatch is 0, and I substract 1.

Reply:
I think I get what you mean, but I don't see how adding it back would change anything